vtysh/vtysh_main.c

```diff
@@ -404,6 +407,10 @@ int main(int argc, char **argv, char **env)
 		case 'w':
 			writeconfig = 1;
 			break;
+		case 'p':
+			inputfile = optarg;
```

Comment:
there's a place where there's a warning about interaction among `-f` and some other options:

`if (inputfile && (writeconfig || boot_flag)) {`

is it worth enhancing that warning?